recipes/siphon/meta.yaml

```diff
@@ -0,0 +1,41 @@
+package:
+    name: siphon
+    version: "0.3.2"
+
+source:
+    fn: siphon-0.3.2.tar.gz
+    url: https://pypi.python.org/packages/source/s/siphon/siphon-0.3.2.tar.gz
+    md5: dd08ff89b1176939d59acae0d544dd81
+
+build:
+    number: 0
+    script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+    build:
+        - python
+        - setuptools
+    run:
+        - python
+        - numpy
```

Comment:
Should there be a version number for `numpy`? I seem to remember that if I didn't have a version, the built packages ended up being built against a particular numpy ABI.